test/arizona_live_view_SUITE.erl

```diff
@@ -64,24 +62,33 @@ mount(Socket) ->
     Socket.
 
 render(Macros) ->
-    ?ARIZONA_LIVEVIEW(Macros, ~s"""
+    arizona_live_view:parse_str(~s"""
     <html>
     <head>
     </head>
     <body>
         Hello, World!
     </body>
     </html>
-    """).
+    """, Macros).
 
 %% --------------------------------------------------------------------
 %% Test cases.
 %% --------------------------------------------------------------------
 
-hello_world(_Config) ->
-    {ok, Resp0} = httpc:request("http://localhost:8080/notfound"),
-    ?ARIZONA_ASSERT_STATUS(404, Resp0),
-    {ok, Resp1} = httpc:request("http://localhost:8080/helloworld"),
-    ?ARIZONA_ASSERT_STATUS(200, Resp1),
-    ?ARIZONA_ASSERT_BODY("Hello, World!", Resp1).
+hello_world(Config) when is_list(Config) ->
+    Resp0 = httpc:request("http://localhost:8080/notfound"),
+    ?assert(is_status(404, Resp0)),
+    Resp1 = httpc:request("http://localhost:8080/helloworld"),
+    ?assert(is_status(200, Resp1)),
+    ?assert(is_body("Hello, World!", Resp1)).
+
+is_body(Pattern, {ok, {{_HttpVersion, _StatusCode, _String}, _HttpHeaders, HttpBodyResult}}) ->
+  nomatch =/= string:find(HttpBodyResult, Pattern);
+is_body(_Pattern, _Result) ->
+  false.
 
+is_status(StatusCode, {ok, {{_HttpVersion, StatusCode, _String}, _HttpHeaders, _HttpBodyResult}}) ->
+  true;
+is_status(_StatusCode, _Result) ->
+  false.
```

Comment:
I moved these here since it's the only place they're used. If we come to a time where we need them elsewhere, we'll move them 😄